File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for i in range(len(commands)):
        stack=[]
        for j in range(commands[i][0]-1,commands[i][1]):
            stack.append(array[j])
        stack.sort()
        answer.append(stack[commands[i][2]-1])
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    vector<int> arr;
    for(const auto& command : commands){
        arr.clear();
        for(int i=command[0]-1;i<command[1];i++){
            arr.push_back(array[i]);
        }
        sort(arr.begin(),arr.end());
        answer.push_back(arr[command[2]-1]);
    }
    
    return answer;
}